manifest.konnector

```diff
@@ -33,8 +32,10 @@
       "verbs": [
         "DELETE",
         "POST",
-        "PUT"
-      ]
+        "PUT",
+        "GET"
```

Comment:
The GET here is needed, without it, the konnector fail in forbidden error

Reply:
Did you find out where the GET is needed please? It should be in konnector-libs, but I'm curious about the reason

Reply:
Yup, GET is needed for saveFiles, as it first GET the file we're trying to save to know if it needs to be saved or not !